lib/openxml/has_attributes.rb

```diff
@@ -27,7 +27,15 @@ def attribute(name, expects: nil, one_of: nil, in_range: nil, displays_as: nil,
       end
 
       def attributes
-        @attributes ||= {}
+        @attributes ||= begin
+          if superclass.respond_to?(:attributes)
+            Hash[superclass.attributes.map { |name, attribute|
+              [ name, attribute.map { |value| value } ]
```

Comment:
Is `attribute.map { |value| value }` an elaborate way of `dup`ing `attribute` so that changes on the child class don't affect the parent class?

Would `attribute.dup` do the same?

Reply:
Yeah, now that you mention it, I think it would. 😝

Reply:
It looks like you use `.dup` in a later commit when you make properties inheritable.